services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy

```diff
@@ -89,6 +104,7 @@ class MetersSpec extends BaseSpecification {
             assert it.meterEntries.size() == 2
             assert it.meterEntries.every { Math.abs(it.rate - (DISCO_PKT_RATE * DISCO_PKT_SIZE) / 1024L) <= 1 }
             assert it.meterEntries.every { it.meterId <= Constants.MAX_DEFAULT_METER_ID }
+            //unable to use #getExpectedBurst. For Centects there's special burst due to KBPS
```

Comment:
Could you please also mention in the comment (or in testcase's narrative) that this path is implemented because Centec switches don't support PKTPS (packets per second) type of meters, which is used for default (discovery) flows?